.circleci/config.yml

```diff
@@ -83,14 +83,25 @@ orbs:
                   exit 0
                 fi
 
-                paths=".circleci circleci <<parameters.paths>>"
+                paths="circleci docs <<parameters.paths>>"
```

Comment:
The solution is just to add docs here and it should be fine. 
Please let `.circleci` It is a valid repo that we need to loop through